casts/texts/package.json

```diff
@@ -0,0 +1,87 @@
+{
+    "name": "@tonlabs/uicast.texts",
```

Comment:
I'm not sure about `uicast.texts`, as UILabel IMO is a core component, and is tightly coupled with current themes (actually it's the only reason to use it over out-of-the-box Text from RN, that it provides correct font and  supports automatic theme switching).

@sertony what do you think?

Reply:
Components from `uicast.texts` (UIExpandableText) uses UILabel from `kit.themes`

Reply:
@savelichalex I honestly don't understand what's your concern about. Is it about the cast name, or `UIExpandableText` component placement in this cast?

I got @SAnatoliiS 's idea about using `UILabel` from `kit.theme` and having this independent component build on top of it.

Reply:
I looked again, and now I'm OK with it :)